ui/app/AppLayouts/Chat/ContactsColumn/ChannelList.qml

```diff
@@ -56,11 +56,11 @@ ScrollView {
         id: channelContextMenu
         width: 175
         subMenuIcons: [
-            {
-                source:  Qt.resolvedUrl("../../../img/bell.svg"),
-                width: 16,
-                height: 16
-            },
+            /* { */
+            /*     source:  Qt.resolvedUrl("../../../img/bell.svg"), */
+            /*     width: 16, */
+            /*     height: 16 */
+            /* }, */
```

Comment:
As discussed offline, the context menu for mute durations is being left out for now